app-backend/db/src/main/scala/tiling.scala

```diff
@@ -0,0 +1,11 @@
+package com.rasterfoundry.database
+
+import geotrellis.layer.ZoomedLayoutScheme
+import geotrellis.proj4.WebMercator
+
+object tiling {
+  val tmsLevels = {
+    val scheme = ZoomedLayoutScheme(WebMercator, 256)
+    for (zoom <- 0 to 64) yield scheme.levelForZoom(zoom).layout
```

Comment:
This is cargo-culted from the `backsplash-core` `BacksplashImage` value. It goes to far higher zooms that we should conceivably need -- I don't recall that we allow above 30 in any UIs, and zoom 60 is an effective pixel size of ~1.4e-13 at the equator (I divided zoom 20 by 2 ^ 40: https://wiki.openstreetmap.org/wiki/Zoom_levels). This is a cheap value to calculate (can try it out in an ammonite shell), it's calculated once, and it gives us some leeway in case we need more zooms than we currently support.